src/text-editor.coffee

```diff
@@ -1074,6 +1074,50 @@ class TextEditor extends Model
       @autoIndentSelectedRows() if @shouldAutoIndent()
       @scrollToBufferPosition([newSelectionRanges[0].start.row - 1, 0])
 
+  # Move any active selections one column to the left.
+  moveSelectionLeft: ->
+    selections = @getSelectedBufferRanges()
+    noSelectionAtStartOfLine = selections.every((selection) ->
+      selection.start.column isnt 0
+    )
+
+    translationDelta = [0, -1]
+    translatedRanges = []
+
+    @transact =>
+      if noSelectionAtStartOfLine
```

Comment:
Yeah, I think if the transaction is a no-op when the selection is at the start/end of line, then we shouldn't create the transaction.